### DISCUSSION TITLE
Launch command in split

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [kovidgoyal/kitty#5068] [Q&A] [answered]
I have several scripts that use fzf and i would like them to be executed in a bottom split (if possible ) , i saw the `launch` command in the config for setting similar functionality to key binds , but could't figure out how to use it as a command. 
`kitty +launch` requires a *.py script from the message i got while trying to run it as a command.

eg.
```
$ m # is an alias to script showing man pages with fzf
........ (window splits)
*fzf script running here*
*when it finishes, execution returns to main window ( in this case the man page opens in the previews window)*

```

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/launch/

**Reply by LamprosPitsillos** [reply]
Thanks but , my problem is how i can use the command outside the key bindings in the config?

**Comment by kovidgoyal** [comment]
On Sun, May 08, 2022 at 08:13:17AM -0700, LamprosPitsillos wrote:
> Thanks but , my problem is how i can use the command outside the key bindings in the config? 


what does using it outside key binding mean? If you want to startup
kitty with some layout use a session file.

**Reply by LamprosPitsillos** [reply]
Let's say i want to run many commands in a split window, like i said i want to use this with some of my scripts. It's not reasonable to have a key binding for every script i have , i only have so many keys. Instead i am looking for a way to launch a split with any command i want. 
Eg. 
Imagine i have 50 scripts:
How would i bind all 50 to key binds, even if i could it wouldn't be desirable.
But if i could some how call launch let's say via kitten i could do this:
```
alias m='kitty +kitten launch man_script.sh'
# This is just an imaginary kitten for my example.

```

**Comment by kovidgoyal** [comment, marked as answer]
See https://sw.kovidgoyal.net/kitty/remote-control/
